README.md

```diff
@@ -21,7 +23,7 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This a migration file it is used to modify the scheme file It can adding and deleting data  from schema. The numbers in the beginning are a timestamp they are use to know which migration runs first.
```

Comment:
:white_check_mark: Yesss. You're the first quiz I've seen that says what the timestamps actually **do**.